openfisca_us/data/geography/create_zip_code_dataset.py

```diff
@@ -0,0 +1,43 @@
+import pandas as pd
```

Comment:
Could you add a README.md to this folder so we know what this script does and when to run it?